lib/capnp_c.h

```diff
@@ -22,6 +28,19 @@
 typedef intmax_t ssize_t;
 #endif
 
+// Macro UNUSED(x) silences compiler warnings about unused parameter or variable `x`.
+#ifndef UNUSED
+#define UNUSED(x) (void)(x)
+#endif
+
+// Cross-platform macro ALIGNED_(x) aligns a struct by `x` bits.
```

Comment:
Actually - the alignment is there because cap'n proto pointers are aligned to 8 bytes... completely forgot that :D ... so it needs to stay. Haven't looked at this in a while :)

Definitely needs to be 8 though, not 64.